packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -467,19 +476,25 @@ describe('plugin-meetings', () => {
           mediaType === AUDIO
             ? meeting.mediaProperties.audioStream?.setServerMuted
             : meeting.mediaProperties.videoStream?.setServerMuted;
-        setMutedSpy =
+        setUserMutedSpy =
           mediaType === AUDIO
-            ? meeting.mediaProperties.audioStream?.setMuted
-            : meeting.mediaProperties.videoStream?.setMuted;
+            ? meeting.mediaProperties.audioStream?.setUserMuted
+            : meeting.mediaProperties.videoStream?.setUserMuted;
 
         clearSpies();
       };
 
       const clearSpies = () => {
         setUnmuteAllowedSpy?.resetHistory();
         setServerMutedSpy?.resetHistory();
-        setMutedSpy?.resetHistory();
+        setUserMutedSpy?.resetHistory();
       };
+
+      const simulateUserMute = (mute) => {
```

Comment:
do we have any tests that simulate system mute?

Reply:
I didn't add any, but I'll take a look to see where it makes sense. Most of the tests end up checking both user mute and system mute anyways, so if anything I can add similar tests but with system mute instead of user mute.